src/appleseed-max-impl/oslutils.cpp

```diff
@@ -353,7 +353,7 @@ void connect_sub_mtl(
         return;
 
     std::string layer_name =
-        make_unique_name(assembly.materials(), wide_to_utf8(mat->GetName()) + "_sub_mat");
+        make_unique_name(assembly.materials(), asf::format("{0}_{1}_{2}", shader_name, mat->GetName(), "sub_mat"));
```

Comment:
"sub_mat" can be part of the format string.

Reply:
such a stupid mistake really :)

Reply:
It's not even really a mistake :)